x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/add_inference_pipeline_flyout.scss

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-.enterpriseSearchInferencePipelineModal {
+.enterpriseSearchInferencePipelineFlyout {
   width: $euiSizeXXL * 20;
```

Comment:
I would suggest not adding a fixed width on the fly out as this is causing some clipping on smaller screen sizes and doesn't allow the fly out to resize naturally. 
![localhost_5601_app_enterprise_search_content_search_indices_search-ikea_pipelines (3) copy](https://user-images.githubusercontent.com/7115017/205716402-fa38d39a-9352-4e1f-82e0-42232547cfba.png)

Can we instead use the `size="l"` prop on the fly out?

Reply:
Good catch, I think this was needed for the modal and I just missed it when migrating to flyout.